frontend/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,223 @@
+import {
+  Box,
+  chakra,
+  Divider,
+  Flex,
+  FlexProps,
+  Stack,
+  Text,
+  Wrap,
+} from '@chakra-ui/react'
+
+import { ThemeColorScheme } from '~/theme/foundations/colours'
+
+import { BxlFacebook } from '~assets/icons/BxlFacebook'
+import { BxlInstagram } from '~assets/icons/BxlInstagram'
+import { BxlLinkedin } from '~assets/icons/BxlLinkedin'
+import { BxlYoutube } from '~assets/icons/BxlYoutube'
+import { ReactComponent as OgpLogoSvg } from '~assets/svgs/brand/ogp-logo.svg'
+import { ReactComponent as OgpLogoFullSvg } from '~assets/svgs/brand/ogp-logo-full.svg'
+import Link from '~components/Link'
+
+const OgpLogoFull = chakra(OgpLogoFullSvg)
+const OgpLogo = chakra(OgpLogoSvg)
+
+const SOCIAL_MEDIA_LINKS = {
+  facebook: 'https://www.facebook.com/opengovsg',
```

Comment:
hmmmmmmmmmm is there a way to do this which will make i18n easier in future? also generally I'm thinking it might be neater to put all the links in one place so it's easy to check that all the URLs in the frontend are up to date

Reply:
I'll make a PR to incorp i18n after this then

Reply:
Actually for links they can probably just live in a constant file

Reply:
maybe we all live in a constant file